### PR TITLE
Additional KDE Frameworks

### PR DESCRIPTION
**Pull request by wrobelda**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

As discussed in https://github.com/Homebrew/discussions/discussions/994, I am submitting a first batch PR for a bunch of KDE Frameworks. These ones are relatively simple, with little to none dependencies on top of the `qt@5`. All of these were previously tested by many users of the [`kde-mac/kde`](https://invent.kde.org/packaging/homebrew-kde) tap, so we're not expecting any functional issues here.

See also the issue tracking this effort on our GitLab instance: https://invent.kde.org/packaging/homebrew-kde/-/issues/23

Remaining tasks:
- [x] Add License info
- [ ] Add aliases
- [ ] Review the `args` – are any of these excessive?

### CODE REVIEW

**Comment by carlocab** [comment]
Thanks for your PR, @wrobelda. Can you split your first commit and squash the rest so that there is one commit per formula file modified (and only one formula file modified per commit)?

This is actually mentioned in the docs [here](https://docs.brew.sh/Formula-Cookbook#push). (I know it's easy to miss; just letting you know it's there for future reference.)

File: Formula/kde-kapidox.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      -DCMAKE_INSTALL_PREFIX=#{libexec}
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_FIND_FRAMEWORK=LAST
+      -DCMAKE_VERBOSE_MAKEFILE=ON
+      -Wno-dev
+    ]
+  end
+
+  def install
+    args = cmake_args
```
Comment (carlocab) [inline Formula/kde-kapidox.rb:40]:
```suggestion
    args = std_cmake_args.reject { |s| s["CMAKE_INSTALL_PREFIX"] }
    args << "-DCMAKE_INSTALL_PREFIX=#{libexec}"
```

File: Formula/kde-kapidox.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", PYTHONPATH: ENV["PYTHONPATH"])
+  end
+
+  test do
+    assert_predicate bin/"kapidox_generate", :exist?
```
Comment (carlocab) [inline Formula/kde-kapidox.rb:63]: Could you propose a test that exercises some functionality of this formula?

**Comment by wrobelda** [comment]
> Thanks for your PR, @wrobelda. Can you split your first commit and squash the rest so that there is one commit per formula file modified (and only one formula file modified per commit)?
> 
> This is actually mentioned in the docs [here](https://docs.brew.sh/Formula-Cookbook#push). (I know it's easy to miss; just letting you know it's there for future reference.)

Ah, my bad, you did ask for it. Let me do that.

File: Formula/kde-kapidox.rb (hunk 2 above)
Comment (wrobelda) [reply Formula/kde-kapidox.rb:63]: @yurikoles are you familiar with kapidox? I am getting the following:
```
~ /opt/homebrew/Cellar/kde-kapidox/5.79.0/bin/kapidox_generate
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/kde-kapidox/5.79.0/libexec/bin/kapidox_generate", line 4, in <module>
    __import__('pkg_resources').run_script('kapidox==5.79.0', 'kapidox_generate')
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3243, in <module>
    def _initialize_master_working_set():
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3226, in _call_aside
    f(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3255, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 568, in _build_master
    ws.require(__requires__)
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/opt/homebrew/lib/python3.9/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'kapidox==5.79.0' distribution was not found and is required by the application
```

I checked `do-caveats.sh` but don't see any hints.

File: Formula/kde-attica.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      find_package(Qt5 REQUIRED Core Network)
+      find_package(KF5Attica REQUIRED)
+    EOS
+
+    args = std_cmake_args
+    args << "-DQt5_DIR=#{Formula["qt@5"].opt_prefix/"lib/cmake/Qt5"}"
+
+    system "cmake", testpath.to_s, *args
+    system "make"
+  end
```
Comment (SMillerDev) [inline Formula/kde-attica.rb:48]: Doesn't this just test of cmake can find it? Instead of functions?

File: Formula/kde-kapidox.rb (hunk 2 above)
Comment (wrobelda) [reply Formula/kde-kapidox.rb]: @carlocab I removed it for now from this batch

File: Formula/kde-attica.rb (hunk 3 above)
Comment (wrobelda) [reply Formula/kde-attica.rb:52]: Find it and load the target, but not whether the libs load properly. I'll try to add more complex tests to these formulas

Comment (wrobelda) [reply Formula/kde-attica.rb:52]: Actually, @SMillerDev, is that necessary? The cookbook uses exactly this type of test as an example of a good one:

```
See cmake for an example of a formula with a good test. The formula writes a basic CMakeLists.txt file into the test directory then calls CMake to generate Makefiles. This test checks that CMake doesn’t e.g. segfault during basic operation.
```

The cmake test is defined as:

```
  test do
    (testpath/"CMakeLists.txt").write("find_package(Ruby)")
    system bin/"cmake", "."
  end
```

**Comment by wrobelda** [comment]
I am gonna file these Formulas separately, after all.

**Pull request closed without merging**